src/Collectors/Callable_/CallUserFuncCollector.php

```diff
@@ -57,11 +56,12 @@ public function processNode(Node $node, Scope $scope): ?array
         }
 
         $callableType = $scope->getType($args[0]->value);
-        if (! $callableType instanceof ConstantArrayType) {
+        if (count($callableType->getConstantArrays()) !== 1) {
```

Comment:
Similar issue with `instanceof ConstantArrayType`

Reply:
same question as in https://github.com/TomasVotruba/unused-public/pull/135/files#r1864884845

discarding when `count() != 1` will lead to false positives (and here we don't handle attributes, so it can easily happen

Reply:
you are right, updated